src/pytorch_lightning/trainer/trainer.py

```diff
@@ -649,6 +647,7 @@ def _call_and_handle_interrupt(self, trainer_fn: Callable, *args: Any, **kwargs:
         """
         try:
             if self.strategy.launcher is not None:
+                self._check_torchdistx_support()
```

Comment:
I don't like having to put this here. Does anybody know a better option?

Reply:
what about a property on the launcher? Something like `meta_supported` or `distx_supported` which we can check?

Not an ideal option either, but couldn't think of anything better...